packages/cdktf/lib/private/fs.ts

```diff
@@ -0,0 +1,48 @@
+import * as fs from "fs";
+import * as path from "path";
+import * as crypto from "crypto";
+
+const HASH_LEN = 32;
+
+// Full implementation at https://github.com/jprichardson/node-fs-extra/blob/master/lib/copy-sync/copy-sync.js
```

Comment:
Any reason to not just use `fs-extra`?

Reply:
I understand we would need to bundle it because of JSII and my understanding was the general course was to rather write the one function we need ourselves than adding a bundled dependency